code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -139,6 +139,8 @@
 		new /obj/item/storage/firstaid/radbgone(src)
 	else
 		new /obj/item/storage/pill_bottle/antirad(src)
+	if(prob(20)) //NSV13 - adding shovels for the meltdowns

```

Comment:
Might need a higher chance

Reply:
Might need a report indicating the case.